src/config/dynamic-theme-fixes.config

```diff
@@ -2240,6 +2240,13 @@ CSS
 
 ================================
 
+doordash.com
+
+INVERT
+.fYPzgJ::after
```

Comment:
It seems to me that's already inverted by default 
![image](https://user-images.githubusercontent.com/25481501/103815007-9579d380-505a-11eb-8f95-940500578248.png)

Reply:
![image](https://user-images.githubusercontent.com/15866023/103830729-3eb7ce00-5049-11eb-8dfa-c1c6cd1dd9d1.png)

The check boxes are fine, the issue only occurs on radio selection, where users are only allowed to select one option.
Example in the screenshot is after the code change.

Reply:
Do you have a example Link to a store who has this as well which item it is?

Reply:
Try this, almost every item on the menu had the radio button.
https://www.doordash.com/store/super-chicken-tysons-17410/en-US

Reply:
Aha thanks!
I already was afright of this and the CSS selector doesn't seem to be working for me due to being it randomized.
An alternative Selector can be:
```CSS
input[kind="RADIO"][id^="Toggle"]::after
```

Reply:
Ah, thanks for the suggestion. This is much safer way of changing it.
Will update the branch.